ArduPlane/tailsitter.cpp

```diff
@@ -268,12 +268,18 @@ void QuadPlane::tailsitter_speed_scaling(void)
         } else {
             // if no airspeed sensor reduce control surface throws at large tilt
             // angles (assuming high airspeed)
-
             // ramp down from 1 to max_atten at tilt angles over trans_angle
             // (angles here are represented by their cosines)
-            constexpr float c_trans_angle = cosf(.125f * M_PI);
-            constexpr float alpha = (1 - max_atten) / (c_trans_angle - cosf(radians(90)));
+
+            // Note that the cosf call will be necessary if trans_angle becomes a parameter
+            // but the C language spec does not guarantee that trig functions can be used
+            // in constant expressions, even though gcc currently allows it.
+            constexpr float c_trans_angle = 0.9238795; // cosf(.125f * M_PI)
+
+            // alpha = (1 - max_atten) / (c_trans_angle - cosf(radians(90)));
```

Comment:
One of these is not like the other.  Did you forget to subtract the `cosf` bit here?

Reply:
cosf(radians(90))=0
unless the 90 degrees becomes a parameter